site/content/en/docs/Getting Started/create-fleetautoscaler.md

```diff
@@ -150,7 +150,7 @@ Last Scale Time has been updated, and a scaling event has been logged.
 Double-check the actual number of game server instances and status by running
 
 ```
-kubectl get gs -o=custom-columns=NAME:.metadata.name,STATUS:.status.state,IP:.status.address,PORT:.status.ports
+kubectl get gs
```

Comment:
Outputs on this page are still old too :cry: 
https://107091d-dot-preview-dot-agones-images.appspot.com/site/docs/getting-started/create-fleetautoscaler/

Reply:
@markmandel I have changed in above files. Let me know if you still see any old output in there. Thanks!